flutter/lib/src/default_integrations.dart

```diff
@@ -8,6 +8,17 @@ import 'package:sentry/sentry.dart';
 import 'sentry_flutter_options.dart';
 import 'widgets_binding_observer.dart';
 
+class WidgetsFlutterBindingIntegration
```

Comment:
I'd add docs

// It is necessary to initialize Flutter method channels so that	
    // our plugin can call into the native code.